test/xpu_api/random/interface_tests/engines_methods.pass.cpp

```diff
@@ -38,6 +40,17 @@ constexpr auto RANLUX24_BASE_MIN = 0;
 constexpr auto RANLUX24_BASE_MAX = 16777215;
 constexpr auto RANLUX24_P = 223;
 constexpr auto RANLUX24_R = 23;
+constexpr auto PHILOX4x32_W   = 32;
+constexpr auto PHILOX4x64_W   = 64;
+constexpr auto PHILOX4x32_N   = 4;
+constexpr auto PHILOX4x64_N   = 4;
+constexpr auto PHILOX4x32_R   = 10;
+constexpr auto PHILOX4x64_R   = 10;
+constexpr auto PHILOX4x32_MIN = 0;
+constexpr auto PHILOX4x32_MAX = (std::numeric_limits<ex::philox4x32::result_type>::max() & 0xFFFFFFFF) - 1;
+constexpr auto PHILOX4x64_MIN = 0;
+constexpr auto PHILOX4x64_MAX = (std::numeric_limits<ex::philox4x64::result_type>::max() & 0xFFFFFFFFFFFFFFFF) - 1;
```

Comment:
do we still need -1 here?

Reply:
I think it is better to do numeric_limits<uint64_fast_t/uint32_fast_t>::max() - 1 here for predefined engines, not to rely on philox api which we are testing in the same file

Reply:
> do we still need -1 here?

I think so. Since method `max()` returns  `(std::numeric_limits<scalar_type>::max() & in_mask) - 1;` and we would like to check its work outside.

Reply:
> I think it is better to do numeric_limits<uint64_fast_t/uint32_fast_t>::max() - 1 here for predefined engines, not to rely on philox api which we are testing in the same file

Yes, that would be better! changed